sass/routes/_activity-log.scss

```diff
@@ -17,7 +17,7 @@
     display: table;
     width: 100%;
     padding: 15px 0;
-    border-top: 1px solid $light-gray;
```

Comment:
Same as these variables, can we safely remove them after using the rgb ones?

Reply:
same as above! :D